iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs

```diff
@@ -47,9 +47,10 @@ public static async Task Main(string[] args)
                 });
             s_logger = loggerFactory.CreateLogger<Program>();
 
+            // Instantiating this seems to do all we need for outputting SDK events to our console log.
```

Comment:
Should we possibly add a comment at line 46 above?  For example:

```csharp
new ColorConsoleLoggerConfiguration
{
    MinLogLevel = LogLevel.Debug,
    // Uncomment the line below to get ALL logs 
    // MinLogLevel = LogLevel.Trace, 
});

or

```csharp
new ColorConsoleLoggerConfiguration
{
    MinLogLevel = LogLevel.Debug, // Set this to LogLevel.Trace to get ALL logs 
});
```

Reply:
yeah, I like that